index.js

```diff
@@ -63,7 +60,16 @@ module.exports = options => {
 	options = options || {};
 
 	if (process.platform === 'win32') {
-		return [];
+		const historyPath = getPath(options);
+		if (historyPath) {
+			return parse(fs.readFileSync(historyPath, 'utf8'));
+		}
+
+		// No histfile found, so spawn a doskey
+		const {stdout} = childProcess.spawnSync('doskey', ['/history'], {
+			encoding: 'utf8'
+		});
+		return stdout.trim().split('\r\n') || [];
```

Comment:
```suggestion
		return stdout.trim().split('\r\n');
```

The last part will never be hit.